pkg/hooks/loader.go

```diff
@@ -182,6 +183,7 @@ func (h *Hook) SetConfigMocks(m []*models.Mock) {
 
 func (h *Hook) UpdateConfigMock(oldMock *models.Mock, newMock *models.Mock) bool {
 	isUpdated := h.configMocks.update(oldMock.TestModeInfo, newMock.TestModeInfo, newMock)
+	h.matchedMocks = append(h.matchedMocks, newMock.Name)
```

Comment:
shouldn't these be inside a mutex because there are many go routine which are deleting and updating mocks..?